app/AcquireIntersonArrayBMode.xml

```diff
@@ -59,5 +59,12 @@
       <minimum>0</minimum>
       <maximum>255</maximum>
     </integer>
+    <double>
+      <name>c</name>
```

Comment:
We could change this from `c` to `sos` -- single character variables are difficult to search for in code.